packages/js/src/plugins/auctionHouseModule/operations/findListings.ts

```diff
@@ -0,0 +1,118 @@
+import { PublicKey } from '@solana/web3.js';
+import { toListingReceiptAccount } from '../accounts';
+import { ListingReceiptGpaBuilder } from '../gpaBuilders';
+import { AuctionHouse, LazyListing, Listing, toLazyListing } from '../models';
+import {
+  Operation,
+  OperationHandler,
+  OperationScope,
+  useOperation,
+} from '@/types';
+import { Metaplex } from '@/Metaplex';
+
+// -----------------
+// Operation
+// -----------------
+
+const Key = 'FindListingsOperation' as const;
+
+/**
+ * Finds Listings by multiple criteria.
+ * Metadata field is ignored whe Mint is provided.
+ *
+ * ```ts
+ * // Find listings by seller and metadata.
+ * const listings = await metaplex
+ *   .auctionHouse()
+ *   .findListings({ auctionHouse, seller, metadata });
```

Comment:
```suggestion
 * Finds Listings by multiple criteria.
 *
 * ```ts
 * // Find all listings in an Auction House.
 * const listings = await metaplex
 *   .auctionHouse()
 *   .findListings({ auctionHouse });
```